pkg/transform/transform.go

```diff
@@ -0,0 +1,48 @@
+package transform
+
+import (
+	pb "github.com/expect-digital/translate/api/translate/translate/v1/gen"
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func MessageFromProto(m *pb.Message) model.Message {
+	return model.Message{
+		ID:      m.Id,
+		Message: m.Message,
+		Fuzzy:   m.Fuzzy,
+	}
+}
+
+func MessagesFromProto(m *pb.Messages) model.Messages {
+	var messagesToAdd []model.Message
```

Comment:
If you know the size in advance, reserve the memory.

```suggestion
	messagesToAdd := make([]model.Message, 0, len(m.Messages))
```